### PR TITLE
Added files for hpe3parprimera helm charts and operators

### PR DESCRIPTION
**Pull request by bhagyashree-sarawate**
Hi Shiva,
Added the files to support hpe3parprimera support for charts as well as operators.
For the document changes i will raise the separate PR after discussion.
Regards
Bhagyashree

### CODE REVIEW

File: helm/values/csi-driver/v1.1.0/values_3parprimera.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Default values for hpe-csi-storage.
```
Comment (shivamerla) [inline helm/values/csi-driver/v1.1.0/values_3parprimera.yaml]: The idea of adding backend_type is to get rid of two different values.yaml files correct?

**Comment by shivamerla** [comment]
@bhagyashree-sarawate changes look good. Let me try it out before we merge.

**Comment by bhagyashree-sarawate** [comment]
> @bhagyashree-sarawate changes look good. Let me try it out before we merge.

@shivamerla , have you tried the changes? any update ?

File: helm/charts/hpe-csi-driver/values.yaml [hunk 2]
```diff
@@ -2,17 +2,12 @@
...
-  # csp-hpe
-  cspImage: hpestorage/nimble-csp:v1.1.0
 
 # image pull policy for all images in csi deployment
 imagePullPolicy: 'Always'
-
-# CSP deployment name
-cspName: nimble-csp
+backendType: nimble
+# CSP deployment name, handled internally according to backendType
```
Comment (shivamerla) [inline helm/charts/hpe-csi-driver/values.yaml:9]: you can remove these commented ones..

File: helm/charts/hpe-csi-driver/values.yaml [hunk 3]
```diff
@@ -21,8 +16,9 @@ secret:
   username: admin
   password: admin
   servicePort: "8080"
-  serviceName: nimble-csp-svc
-  name: nimble-secret
+# The following parameters are handled internally depending on the backendType
```
Comment (shivamerla) [inline helm/charts/hpe-csi-driver/values.yaml:19]: same here..

**Pull request merged**